packages/react-ui-ag/src/Filters/RatingFilterCard.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+import isEqual from 'lodash/isEqual'
+import { RadioGroup } from '@rentpath/react-ui-core'
+import autobind from 'autobind-decorator'
+import FilterCard from './FilterCard'
+
+@themed(/^RatingFilterCard/)
+export default class RatingFilterCard extends Component {
+  static propTypes = {
+    title: PropTypes.node,
+    description: PropTypes.node,
+    className: PropTypes.string,
+    theme: PropTypes.object,
+    fields: PropTypes.arrayOf(PropTypes.shape({
+      label: PropTypes.string,
+      value: PropTypes.string,
+    })),
+  }
+
+  static defaultProps = {
+    title: 'View 4+ stars units in Atlanta',
```

Comment:
Just a slight nit-pick - I'm ok if there is consensus not to change it, but maybe we should take off Atlanta as a default? Probably just `View 4+ star units'?

Reply:
Doesn't matter. I left it there so one can see what it should look like but in the real app I think ag.js would pass the title dynamically.